qiskit/test/mock/fake_openpulse_2q.py

```diff
@@ -78,6 +78,48 @@ def __init__(self):
                           'r': 0.02,
                           'alpha0': -0.33,
                           'alpha1': -0.33}
+            },
+            channels={
+                'acquire0': {
+                    'operates': {'qubits': [0]},
+                    'purpose': 'acquire',
+                    'type': 'acquire'
+                },
+                'acquire1': {
+                    'operates': {'qubits': [1]},
+                    'purpose': 'acquire',
+                    'type': 'acquire'
+                },
+                'd0': {
+                    'operates': {'qubits': [0]},
+                    'purpose': 'drive',
+                    'type': 'drive'
+                },
+                'd1': {
+                    'operates': {'qubits': [1]},
+                    'purpose': 'drive',
+                    'type': 'drive'
+                },
+                'u0': {
```

Comment:
really minor, but could this `u0` go with `u1`?

Reply:
Fixed in [5f18c70](https://github.com/Qiskit/qiskit-terra/pull/4085/commits/5f18c70f7129702cfbb3f92bf361ee5090e69513)